### PR TITLE
onecommander: add version 3.4.2.0

### LINKED ISSUE #6344: [Request] Add One Commander
Add One Commander (https://onecommander.com/) to the bucket if possible. Seems like a promising file manager although I dislike that it seems not to be open source. Double Commander could also be a good idea.

### PR DESCRIPTION
**Pull request by igaryhe**
Closes #6344

Relates to #XXXX

### CODE REVIEW

File: bucket/onecommander.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "homepage": "http://onecommander.com/",
+    "description": "a file manager for Microsoft Windows with improved functionality, re-imagined user interface and experience of navigating filesystem and manipulating files.",
+    "version": "3.4.2.0",
+    "license": "freeware",
```
Comment (rashil2000) [inline bucket/onecommander.json:5]:
```suggestion
    "version": "3.4.2.0",
    "description": "A file manager with improved functionality, re-imagined user interface and experience of navigating filesystem and manipulating files.",
    "homepage": "https://onecommander.com/",
    "license": "freeware",
```

File: bucket/onecommander.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "shortcuts": [
+        [
+            "OneCommander.exe",
+            "OneCommander"
+        ]
+    ],
+    "checkver": {
+        "url": "http://onecommander.com/blog/",
+        "regex": "Update ([\\d.]+)"
+    },
```
Comment (rashil2000) [inline bucket/onecommander.json:18]: This will not work

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/onecommander.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "3.4.2.0",
+    "description": "A file manager with improved functionality, re-imagined user interface and experience of navigating filesystem and manipulating files.",
+    "homepage": "https://onecommander.com/",
+    "license": "Freeware",
+    "url": "https://onecommander.com/OneCommander3.4.2.0.zip",
+    "hash": "6f9aba6c53067e1c9cdf5d24daa139fb60bacd52a4d74e174f0245a959bf19be",
+    "bin": "OneCommander.exe",
```
Comment (rashil2000) [inline bucket/onecommander.json:8]:
```suggestion
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]
Add persist for "Settings" folder

**Comment by rashil2000** [comment]
Use https everywhere

**Review by rashil2000 — APPROVED** [review]

File: bucket/onecommander.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+    "version": "3.4.2.0",
+    "description": "A file manager with improved functionality, re-imagined user interface and experience of navigating filesystem and manipulating files.",
+    "homepage": "https://onecommander.com/",
+    "license": "Freeware",
```
Comment (issaclin32) [inline bucket/onecommander.json:5]:
```suggestion
    "license": {
        "identifier": "Freeware",
        "url": "http://www.onecommand.com/terms-of-use"
    },
```

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**